src/site/content/en/blog/one-line-layouts/index.md

```diff
@@ -0,0 +1,316 @@
+
+First specify `grid` as the `display` method, and then write `place-items: center` on the same element. `place-items` is a shorthand to set both `align-items` and `justify-items` at once. By setting it to `center`, both `align-items` and `justify-items` are set to  `center`.
+
+```css/2
+.parent {
+  display: grid;
+  place-items: center;
+}
+```
+
+This enables the content to be perfectly centered within the parent, regardless of intrinsic size.
+
+## 02. The Deconstructed Pancake: `flex: <grow> <shrink> <baseWidth>`
+
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/02-deconstructed-pancake-1.mp4'>
+  </video>
+</figure>
+
+Next we have the deconstructed pancake! This is a common layout for marketing sites, for example, which may have a row of 3 items, usually with an image, title, and then some text, describing some features of a product. On mobile, we'll want those to stack nicely, and expand as we increase the screen size. 
+
+By using Flexbox for this effect, you won't need media queries to adjust the placement of these elements when the screen resizes.
+
+The [`flex`](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) shorthand stands for: `flex: <flex-grow> <flex-shrink> <flex-basis>`.
+
+Because of this, if you want your boxes to fill out to their `<flex-basis>` size, shrink on smaller sizes, but not *stretch* to fill any additional space, write: `flex: 0 1 <flex-basis>`, setting the `<flex-grow>` to 0. In this case, your `<flex-basis`> is `'150px'` so it looks like:
+
+```css/5
+.parent {
+  display: flex;
+}
+
+.child {
+  flex: 0 1 150px;
+}
+```
+
+If you *do* want the boxes to stretch and fill the space as they wrap to the next line, set the `'flex-grow'` to `'1'`, so it would look like:
+
+```css/5
+.parent {
+  display: flex;
+}
+
+.child {
+  flex: 1 1 150px;
+}
+```
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/02-deconstructed-pancake-2.mp4'>
+  </video>
+</figure>
+
+Now, as you increase or decrease the screen size,  these flex items both shrink and grow.
+
+## 03. Sidebar Says: `grid-template-columns: minmax(<min>, <max>) ...)`
+
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/03-sidebar-says.mp4'>
+  </video>
+</figure>
+
+This demo takes advantage of the [minmax](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax) function for grid layouts. What we're doing here is setting the minimum sidebar size to be `'150px`', but on larger screens, letting that stretch out to `'25%'`. The sidebar will always take up `'25%'` of its parent's horizontal space until that `'25%'` becomes smaller than `'150px'`.
```

Comment:
```suggestion
This demo takes advantage of the [minmax](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax) function for grid layouts. What we're doing here is setting the minimum sidebar size to be `150px`, but on larger screens, letting that stretch out to `25%`. The sidebar will always take up `25%` of its parent's horizontal space until that `25%` becomes smaller than `150px`.
```

@una there's still single quotes around all of these values. Please remove them throughout the entire post. Ping me on Chat or set up a videochat with me if it's not clear why I want this changed.